Release notes for max auth age password policy (#24889)
keycloak/keycloak: docs/documentation/release_notes/topics/23_0_0.adoc

Keycloak now features `http-max-queued-requests` option to allow proper rejecting of incoming requests under high load.
For details refer to the https://www.keycloak.org/server/configuration-production[production guide].

= Password policy for specify Maximum authentication time

> Keycloak supports new password policy, which allows to specify the maximum age of an authentication with which a password may be changed by user without re-authentication.
> When this password policy is set to 0, user will be required to re-authenticate always when he wants to change his passwords in the account console (or by other means).
> It is also possible to specify some shorter or bigger value than default 5 minutes. Thanks to https://github.com/thomasdarimont[Thomas Darimont] for the contribution.
> 

Reviewer: ```suggestion
Keycloak supports new password policy, which allows to specify the maximum age of an authentication with which a password may be changed by user without re-authentication.
When this password policy is set to 0, the user will be required to re-authenticate to change the  password in the Account Console or by other means.
You can also specify a lower or higher value than the default value of 5 minutes. Thanks to https://github.com/thomasdarimont[Thomas Darimont] for the contribution.

```
